nixos/modules/services/games/asphyxia-core.nix

```diff
@@ -0,0 +1,158 @@
+{ config, lib, options, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.asphyxia-core;
+  opt = options.services.asphyxia-core;
+  configAsIni = generators.toINIWithGlobalSection {} {
+    globalSection = {
+          port = cfg.port;
+          bind = cfg.bind;
+          ping_ip = cfg.pingIp;
+          matching_port = cfg.matchingPort;
+          allow_register = cfg.allowRegister;
+          maintenance_mode = cfg.maintenanceMode;
+          enable_paseli = cfg.enablePaseli;
+    };
+    sections = {};
+  };
+
+  configIniFile = pkgs.writeText "config.ini" configAsIni;
+
+  configDir = pkgs.runCommand "createConfigDir" {} ''
+    mkdir -p $out
+    # create symlink to config file
+    ln -s ${configIniFile} $out/config.ini
+    cp -R ${corePlugins} $out/plugins
+  '';
+
+  startSh = pkgs.writeShellScript "asphyxia.sh" ''
+    CONFIG_DIR=${configDir}
+    echo "starting asphyxia in $CONFIG_DIR"
+    exec -a $CONFIG_DIR/asphyxia-core ${cfg.package}/bin/asphyxia-core -d /var/lib/asphyxia-core
+  '';
+
+  corePlugins = pkgs.fetchgit {
+    url = "https://github.com/asphyxia-core/plugins";
+    rev = "81630a86a299fd278b86b04dd94ece0b36525b2f";
+    sha256 = "sha256-HNtVzlnGbOqXWVbyRqV4sJWOGk9srARPx7vAjmAh4QY=";
+  };
```

Comment:
I can use fetchFromGithub, but the package has no use for these.
Due to a quirk of how Asphyxia is built, it will look for its own execution path in argv[0] and use that to determine the folder from which to load plugins & config files. Because of this, it's useless to place the plugins in the nix store, since that's read-only - which means users won't be able to put a config file in there, or customize any of the options.